src/GeneratedHydrator/ClassGenerator/HydratorGenerator.php

```diff
@@ -49,24 +52,15 @@ class HydratorGenerator implements HydratorGeneratorInterface
      */
     public function generate(ReflectionClass $originalClass) : array
     {
-        $builder   = new ClassBuilder();
+        $class = new Class_($originalClass->getShortName());
 
-        $ast = $builder->fromReflection($originalClass);
+        $ast = array($class);
```

Comment:
Most of @Ocramius' code do not use the short array syntax, I did this on purpose to avoid frustrating him. It's actually never wrote with short array syntax in the whole library. I'm just keeping consistency with the existing codebase.